include/items.php

```diff
@@ -939,6 +939,10 @@ function item_store($arr, $force_parent = false, $notify = false, $dontcache = f
 		call_hooks('post_remote', $arr);
 	}
 
+	// This array field is used to trigger some automatic reactions
```

Comment:
Don't abuse single-line comments for multiple line comments! Use this instead:

```
/*
 * This array field is used to trigger some automatic reactions.
 * It is mainly used in the "post_local" hook.
 */
```

And please include punctuation characters.

Else, it looks okay.